docs/pages/docs/text-field.mdx

```diff
@@ -146,7 +182,7 @@ FTextField.multiline(
     <Widget name='text-field' variant='form' height={550} query={{}}/>
   </Tabs.Tab>
   <Tabs.Tab>
-    ```dart {22-26, 28-32}
+    ```dart {9-11, 20-21, 26-27, 31-36, 38-42, 46-52}
```

Comment:
Out of curiosity, what does lines 20-21 highlight?

Reply:
I thought of just including all the important lines relating to the form. 20-21 is the disposal methods for the `TextEdittingController`.